Src/IronPython/Modules/_fileio.cs

```diff
@@ -442,7 +353,7 @@ public FileIO(CodeContext/*!*/ context, string name, string mode = "r", bool clo
 
                 var span = pythonBuffer.AsSpan();
                 for (int i = 0; i < span.Length; i++) {
-                    int b = _readStream.ReadByte();
+                    int b = _streams.ReadStream.ReadByte();
```

Comment:
Unrelated to the changes, but I sure hope nothing calls this code path. 😅

Reply:
I sure hope too! 😃 

I have some ideas for optimizing reading/writing, which I will submit in a separate PR.